terraform/azure/mds-existing-vnet/README.md

```diff
@@ -0,0 +1,177 @@
+                //  subscription_id = var.subscription_id
+                //  client_id = var.client_id
+                //  client_secret = var.client_secret
+                //  tenant_id = var.tenant_id
+                
+                   features {}
+                }
+            
+        b. In the terraform.tfvars file leave empty double quotes for client_secret, client_id , tenant_id and subscription_id variables:
+        
+                client_secret                   = ""
+                client_id                       = ""
+                tenant_id                       = ""
+                subscription_id                 = "" 
+        
+    <br>2. Using **az** commands from a command-line:
+    - Run  **az login** command 
+    - Sign in with your account credentials in the browser
+    - [Accept Azure Marketplace image terms](https://docs.microsoft.com/en-us/cli/azure/vm/image/terms?view=azure-cli-latest) by running:
+     <br>**az vm image terms accept --urn publisher:offer:sku:version**, where:
+        - publisher = checkpoint;
+        - offer = vm_os_offer (see accepted values in the table below);
+        - sku = vm_os_sku (see accepted values in the table below);
+        - version = latest<br/>
+    <br>Example:<br>
+    az vm image terms accept --urn checkpoint:check-point-cg-r8040:sg-byol:latest
+    
+    - In the terraform.tfvars file leave empty double quotes for client_secret, client_id and tenant_id variables. 
+ 
+- Fill all variables in the /terraform/azure/mds-existing-vnet/terraform.tfvars file with proper values (see below for variables descriptions).
+- From a command line initialize the Terraform configuration directory:
+
+        terraform init
+- Create an execution plan:
+ 
+        terraform plan
+- Create or modify the deployment:
+ 
+        terraform apply
+
+### terraform.tfvars variables:
+ | Name          | Description   | Type          | Allowed values |
+ | ------------- | ------------- | ------------- | -------------  |
+ | **client_secret** | passwordThe client secret of the Service Principal used to deploy the solution | string |
+ |  |  |  |  |  |
+ | **client_id** | The client ID of the Service Principal used to deploy the solution | string |
+ |  |  |  |  |  |
+ | **tenant_id** | The tenant ID of the Service Principal used to deploy the solution | string |
+ |  |  |  |  |  |
+ | **subscription_id** | The subscription ID is used to pay for Azure cloud services | string |
+ |  |  |  |  |  |
+ | **source_image_vhd_uri** | The URI of the blob containing the development image. Please use noCustomUri if you want to use marketplace images. | string | 
+ |  |  |  |  |  |
+ | **resource_group_name** | The name of the resource group that will contain the contents of the deployment | string | Resource group names only allow alphanumeric characters, periods, underscores, hyphens and parenthesis and cannot end in a period |
+ |  |  |  |  |  |
+ | **mds_name** | MDS name. | string |  
```

Comment:
MDS name**.**

Please decide if you want to use dots or not, and align for all.